Index: etc/network/interfaces
===================================================================
--- etc.orig/network/interfaces
+++ etc/network/interfaces
@@ -54,11 +54,11 @@ iface tiwlan0 inet dhcp
 iface atml0 inet dhcp
 
 # Wired or wireless interfaces
-auto eth0
-iface eth0 inet dhcp
-        pre-up /bin/grep -v -e "ip=[0-9]\+\.[0-9]\+\.[0-9]\+\.[0-9]\+" /proc/cmdline > /dev/null
+#auto eth0
+#iface eth0 inet dhcp
+#        pre-up /bin/grep -v -e "ip=[0-9]\+\.[0-9]\+\.[0-9]\+\.[0-9]\+" /proc/cmdline > /dev/null
 
-iface eth1 inet dhcp
+#iface eth1 inet dhcp
 
 # Ethernet/RNDIS gadget (g_ether)
 # ... or on host side, usbnet and random hwaddr
@@ -82,3 +82,11 @@ iface usbf inet static
 # Bluetooth networking
 iface bnep0 inet dhcp
 
+auto eth0
+iface eth0 inet static
+address 192.168.9.13
+netmask 255.255.255.0
+network 192.168.9.0
+gateway 192.168.9.2
+
+
